Update managing hostname page to include new dns details (#2928)
umbraco/UmbracoDocs: Umbraco-Cloud/Set-Up/Manage-Hostnames/index.md

You can bind any hostname to your project environments. Keeping in mind that the hostname will need to have a DNS entry so that it resolves to the Umbraco Cloud service.

> Once you add a hostname to one of your environments make sure to update the hostname DNS entry to resolve to the umbraco.io service. We recommend setting an CNAME record for your root hostname (e.g. mysite.s1.umbraco.io) pointing to **dns.umbraco.io** or an A record using IP 104.19.191.28 or 104.19.208.28. Check with your DNS host or hostname registrar for details on how to configure this for your hostnames.

Reviewer: ```suggestion
Once you add a hostname to one of your environments make sure to update the hostname DNS entry to resolve to the umbraco.io service. We recommend setting a CNAME record for your root hostname (e.g. mysite.s1.umbraco.io) pointing to **dns.umbraco.io** or an A record using an IP, either `104.19.191.28` or `104.19.208.28`. Check with your DNS host or hostname registrar for details on how to configure this for your hostnames.
```

:::warning

> Adding an A-Record to the static ip is only recommended when setting up a CNAME record is not an option. The static IP is highly volatile towards changes to the Umbraco Cloud infrastructure and as such it may change.

Reviewer: ```suggestion
Adding an A-Record to the static IP is only recommended when setting up a CNAME record is not an option. The static IP is highly volatile towards changes to the Umbraco Cloud infrastructure and as such, it may change.
```

Once you've assigned a hostname to your Umbraco Cloud environment, you may want to hide the default umbraco.io URL (e.g. *snoopy.s1.umbraco.io*). We've created a rewrite rule for this purpose - find it in the [Rewrites on Cloud](Rewrites-on-Cloud/#hiding-the-default-umbraco-io-url) article.

### Automatic TLS (HTTPS)

> All hostnames added to an Umbraco Cloud project's environment will get a TLS (HTTPS) certificate added by default. The certificate is issued by Cloudflare and valid for 1 year after which it will be automatically renewed. Everything around certificates and renewals is handled for you, so the only thing you need to worry about is updating the DNS records for your hostname.

Reviewer: ```suggestion
All hostnames added to a Umbraco Cloud project's environment will get a TLS (HTTPS) certificate added by default. The certificate is issued by Cloudflare and valid for 1 year after which it will be automatically renewed. Everything around certificates and renewals is handled for you and you only need to make sure that the DNS records are configured according to our recommendations listed above.
```
